dist/common/scripts/scylla_util.py

```diff
@@ -300,19 +301,25 @@ def is_recommended_instance(self):
         if not self.is_unsupported_instance_class() and self.is_supported_instance_class() and self.is_recommended_instance_size():
             # at least 1:2GB cpu:ram ratio , GCP is at 1:4, so this should be fine
             if self.cpu/self.memoryGB < 0.5:
-              # 30:1 Disk/RAM ratio must be kept at least(AWS), we relax this a little bit
-              # on GCP we are OK with 50:1 , n1-standard-2 can cope with 1 disk, not more
               diskCount = self.nvmeDiskCount
               # to reach max performance for > 16 disks we mandate 32 or more vcpus
               # https://cloud.google.com/compute/docs/disks/local-ssd#performance
               if diskCount >= 16 and self.cpu < 32:
+                  logging.warning("This machine doesn't have enough CPUs for allocated number of NVMEs (at least 32 cpus for >=16 disks). Performance will suffer.")
                   return False
               diskSize= self.firstNvmeSize
               if diskCount < 1:
                   return False
+              max_disktoramratio = 60
+              # 30:1 Disk/RAM ratio must be kept at least(AWS), we relax this a little bit
+              # on GCP we are OK with {max_disktoramratio}:1 , n1-standard-2 can cope with 1 disk, not more
               disktoramratio = (diskCount*diskSize)/self.memoryGB
-              if (disktoramratio <= 50) and (disktoramratio > 0):
-                  return True
+              if (disktoramratio > max_disktoramratio):
+                  logging.warning(f"Instance disk-to-RAM ratio is {disktoramratio}, which is higher than the recommended ratio {max_disktoramratio}. Performance may suffer.")
+                  return False
```

Comment:
So do we really want to return `False` here? You moved the ratio limit to 60, but that's arbitrary (61 could be fine just as well). So my thinking was that we turn this error into a warning instead (so always return `True`).

If you really insist returning `False` here, please argue your case, and let's move forward with one of the two solutions.

Reply:
up to 100 should be fine so I do agree with Avi

but my argument is that I'd rather let iotune decide on actual parameters than hardcoding something that might not work